package.json

```diff
@@ -66,7 +66,7 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "flow-bin": "^0.158.0",
+    "flow-bin": "^0.159.0",
```

Comment:
Ditto.  Is this change really required in this PR?

Reply:
Reverted, thanks.